shared/actions/devices.js

```diff
@@ -105,6 +109,8 @@ const clearBadgesAfterNav = (state, action) => {
   }
 }
 
+const clearNavBadges = state => RPCTypes.deviceDismissDeviceChangeNotificationsRpcPromise().catch(logError)
```

Comment:
This doesn't seem right. It immediately dismisses the `NEW` tag on any devices when you view the list. I think git is supposed to clear the badge on the tab when you nav in but leave it on the row (I can't get a git badge to show so can't test it)